lib/eblox/data/post.ex

```diff
@@ -71,13 +71,19 @@ defmodule Eblox.Data.Post do
   def on_transition(:read, :parse, nil, %{content: content} = payload) do
     case EbloxParser.parse(content) do
       {"", %Md.Parser.State{} = parsed} ->
-        {html, properties} = Md.generate(parsed, walker: &Walker.prewalk/2, format: :none)
+        {initial_properties, payload} = Map.pop(payload, :properties, %{})
+        {html, parsed_properties} = Md.generate(parsed, walker: &Walker.prewalk/2, format: :none)
+
+        properties =
+          @default_properties
+          |> Map.merge(initial_properties)
+          |> Map.merge(parsed_properties)
```

Comment:
[`Map.merge/2`](https://hexdocs.pm/elixir/Map.html#merge/2) does not do deep merge, [`Map.merge/3`](https://hexdocs.pm/elixir/Map.html#merge/3) does and I bet we want at least tags to be _added_ not overwritten.
```suggestion
          |> Map.merge(initial_properties, fn _, v1, v2 -> v1 ++ v2 end)
          |> Map.merge(parsed_properties, fn _, v1, v2 -> v1 ++ v2 end)
```

or something more sophisticated. The format of accepted parameters and what would happen to them should be documented.